### ISSUE TITLE
win32_system_backdrop Acrylic not working

### ISSUE THREAD

**Issue by Lesnock** [wez/wezterm#4145]
### What Operating System(s) are you seeing this problem on?

Windows

### WezTerm version

20230806-225328-bf172634

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

Yes, and I updated the version box above to show the version of the nightly that I tried

### Describe the bug

I set `win32_system_backdrop` to "Acrylic", but the terminal does not get transparent, even with `window_background_opacity` set to `0`.
Besides that, weird border appears and the terminal has so much lag when I try to move it with the mouse (when Acrylic is enabled).

Opacity works great without the acrylic enabled.

### To Reproduce

Set win32_system_backdrop to Acrylic with opacity minor than 1.

### Configuration

```lua
local wezterm = require 'wezterm'
local config = {}

-- Appearance
config.font = wezterm.font 'JetBrainsMono Nerd Font Mono'
config.font_size = 16
config.line_height = 1.2
config.window_background_opacity = 0.90

-- Theme
config.color_scheme = 'Catppuccin Macchiato'

-- Enable Programmer Background Image
config.window_background_gradient = {
    colors = {
        '#24273a',
    },
}
config.background = {
    {
        source = { File = 'C:/Program Files/WezTerm/hacker.gif' },
        opacity = 0.02,
        width = 243,
        height = 177,
        repeat_x = "NoRepeat",
        repeat_y = "NoRepeat",
        vertical_align = 'Bottom',
        horizontal_align = 'Right',
        vertical_offset = '-30'
    },
}
config.win32_system_backdrop = 'Acrylic'

config.window_padding = {
    left = 20,
    right = 20,
    top = 20,
    bottom = 20,
}

-- Start
config.default_domain = 'WSL:Ubuntu-22.04'
config.initial_rows = 16

-- Tabs
config.enable_tab_bar = true
config.hide_tab_bar_if_only_one_tab = true
local act = wezterm.action
config.keys = {
  -- Create a new tab in the default domain
  { key = 't', mods = 'SHIFT|ALT', action = act.SpawnTab 'DefaultDomain' },
}
wezterm.on("format-tab-title", function(tab, tabs, panes, config, hover, max_width)
    local title = string.format('Tab %s', tab.tab_index + 1)
    
	return {
		{ Text = title },
	}
end)
config.window_close_confirmation = 'NeverPrompt'

return config
```

### Expected Behavior

Expect the terminal to get blurry.

**Comment by Lesnock** [comment]
My Windows build: 19045.3324

**Comment by charliemonnone** [comment]
Having the same issue, also on Windows 10.

**Comment by RichardEpure** [comment]
Do you have "Transparency effects" in the windows settings turned on? It won't work if that's not the case. 

![image](https://github.com/wez/wezterm/assets/43828146/7e676cd2-c108-457b-bb06-4fb5c0c4bff9)

**Comment by charliemonnone** [comment]
> Do you have "Transparency effects" in the windows settings turned on? It won't work if that's not the case.
> 
> ![image](https://private-user-images.githubusercontent.com/43828146/297524418-7e676cd2-c108-457b-bb06-4fb5c0c4bff9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU2ODQ0MDUsIm5iZiI6MTcwNTY4NDEwNSwicGF0aCI6Ii80MzgyODE0Ni8yOTc1MjQ0MTgtN2U2NzZjZDItYzEwOC00NTdiLWJiMDYtNGZiNWMwYzRiZmY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE5VDE3MDgyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ5YzUwNDdiZTlkZmU1M2QxMTI2NDBkZDBmZmNhZDAyOWEyYjQ5NzE2Njg5YzNkOGU3ZmFkZTBmODc4MGFmNTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.k0dUgE2whjKaHXgQec5RHSvrLH5rdInn07NwwIyK6Mc)

Yes, though on Windows 10 this setting is under "Colors", not Accessibility.

**Comment by JacobLChrzanowski** [comment]
I'm having the same issue across all 3 blur modes on Windows 10 Pro, 22H2, build 119045.3693

**Comment by Lavoista** [comment]
same issue here, I have transparency on in windows settings, in wezterm I have
config.window_background_opacity = 0
config.win32_system_backdrop = 'Acrylic'

No blur effect :(

**Comment by RevinderDev** [comment]
Same issue here.

version: wezterm 20240203-110809-5046fc22
Windows 10 Pro 10.0.19045 Build 19045

Not sure if related but even opacity setting, doesn't actually make the background transparent at all (even when set to 0). Just changes color to black. This could be something silly I've set in Windows ages ago, just not sure what.

**Comment by nikolas-sturm** [comment]
Originally posted here: https://github.com/wezterm/wezterm/issues/4881#issuecomment-2976014974

I also had this problem and none of the solutions mentionend in #4145 or #4881 worked for me. I did a bit of experimenting based on @ppaauuoo's answer and found that the setting that made it work for me was "Vulkan/OpenGL present method" (In NVIDIA Control Panel -> 3D Settings). This was globally set to "Prefer layered on DXGI Swapchain" which causes the black backdrop for me. I manually added wezterm to individual programs and set the method to "Prefer native". This gave me back the mica backdrop and made transparency work again.
As reference i am using:
```lua
config.front_end = "WebGpu"
config.webgpu_power_preference = "HighPerformance"
config.window_background_opacity = 0
config.win32_system_backdrop = "Mica"
```
I also tested with `front_end = "OpenGL"` and the method also fixed it in this case.

**This method also worked for "Acrylic" in my case!**

**Comment by jayhathaway** [comment]
> Originally posted here: [#4881 (comment)](https://github.com/wezterm/wezterm/issues/4881#issuecomment-2976014974)
> 
> I also had this problem and none of the solutions mentionend in [#4145](https://github.com/wezterm/wezterm/issues/4145) or [#4881](https://github.com/wezterm/wezterm/issues/4881) worked for me. I did a bit of experimenting based on [@ppaauuoo](https://github.com/ppaauuoo)'s answer and found that the setting that made it work for me was "Vulkan/OpenGL present method" (In NVIDIA Control Panel -> 3D Settings). This was globally set to "Prefer layered on DXGI Swapchain" which causes the black backdrop for me. I manually added wezterm to individual programs and set the method to "Prefer native". This gave me back the mica backdrop and made transparency work again. As reference i am using:
> 
> config.front_end = "WebGpu"
> config.webgpu_power_preference = "HighPerformance"
> config.window_background_opacity = 0
> config.win32_system_backdrop = "Mica"
> I also tested with `front_end = "OpenGL"` and the method also fixed it in this case.
> 
> **This method also worked for "Acrylic" in my case!**

This fixed it for me too.
